addons/web/static/src/views/view_dialogs/export_data_dialog.xml

```diff
@@ -13,8 +13,8 @@
 
     <t t-name="web.ExportDataItem" owl="1">
         <div t-att-data-field_id="props.field.name" t-attf-class="o_export_tree_item cursor-pointer position-relative ps-4 {{ state.isExpanded ? 'o_expanded mb-2' : '' }}" role="treeitem" t-on-click.stop="onClick" t-on-dblclick="(ev) => !props.isFieldExpandable(props.field) and !this.isFieldSelected(props.field.name) and this.props.onAdd(ev)">
-            <span t-if="props.isFieldExpandable(props.field)" t-attf-class="o_expand_parent d-inline-block position-absolute small fa {{ state.isExpanded ? 'fa-chevron-down' : 'fa-chevron-right' }}" role="img" aria-label="Show sub-fields" title="Show sub-fields" />
-            <div t-attf-class="o_tree_column d-flex justify-content-between align-items-center {{ props.field.required ? 'fw-bolder' : ''}}">
+            <div t-attf-class="o_tree_column justify-content-between align-items-center {{ props.field.required ? 'fw-bolder' : ''}}">
```

Comment:
Thanks for the fix :) You can also remove the `justify-content-between` and `align-items-center` classes (those are flex related)

Reply:
Done. Thanks for the tips